price_parser/parser.py

```diff
@@ -205,9 +205,9 @@ def extract_price_text(price: str) -> Optional[str]:
 _search_decimal_sep = re.compile(r"""
 \d           # at least one digit (there can be more before it)
 ([.,€])      # decimal separator
-(?:          # 1,2 or 4 digits. 3 digits is likely to be a thousand separator.
+(?:          # 1,2 or 4+ digits. 3 digits is likely to be a thousand separator.
    \d{1,2}|
-   \d{4}
+   \d{4,8}
```

Comment:
I'm curious why 8 as the limit? I assume these long prices come from someone doing some calculations on a price as a float64 number, in which case it's representation can have at least 15 digits?

Reply:
Yeah, this is a part I'm not sure of. I set a limit to 8 just in case, as I haven't seen large float-point prices before, to be on a safe side and avoid over-matching. But actually you're right - if this regexp doesn't detect decimal point, it is just stripped out, which is not what we want. I'll send another pull request.